fastparquet/api.py

```diff
@@ -122,36 +122,35 @@ def row_group_filename(self, rg):
         else:
             return self.fn
 
-
     def read_row_group_file(self, rg, columns, categories, index=None,
-                            assign=None):
+                            assign=None, mr_times=[]):
```

Comment:
Yes, there are too many keywords, a refactor is probably warranted, but I think that should be a new PR.
On the name, I explicitly want MR in there, because this is *not* the "interaval" type listed here https://github.com/Parquet/parquet-format/blob/master/LogicalTypes.md, which is also int96.